src/scanner-cli.ts

```diff
@@ -94,9 +103,22 @@ export async function downloadScannerCli(properties: ScannerProperties): Promise
   // Create parent directory if needed
   await fsExtra.ensureDir(installDir);
 
+  // Add basic auth credentials when used in the UR
+  let overrides: AxiosRequestConfig | undefined;
+  if (scannerCliUrl.username && scannerCliUrl.password) {
+    overrides = {
+      headers: {
+        Authorization:
+          'Basic ' +
+          Buffer.from(`${scannerCliUrl.username}:${scannerCliUrl.password}`).toString('base64'),
+      },
+    };
+  }
```

Comment:
TODO: for non- SQ, drop the default `Authorization` header

Reply:
we could explicitly use axios without our instance here to avoid some of the  authorization issue